oat-sa/extension-tao-itemqti
# Hotfix/ta0 2323 pr

how to test:
* create a latex expression with a special char (*, \leq, ...)
* switch to mathml mode and check that the converted code doesn't contains any comment
* broken items (with &lt;!-- * --&gt; in a mathml code) can be fixed by open the item, saving it, and refreshing the page
* entering comments directly in mathml editor should not break rendering nor item

there might still be others ways to break this...
